modules/ROOT/pages/_partials/app-security/setting-up-the-app-security-sdk.adoc

```diff
@@ -0,0 +1,43 @@
+// For more information, see: https://redhat-documentation.github.io/modular-docs/
+
+include::configuring-dev-env-app-security.adoc[]
+
+== Setting up {app-security-service} service SDK
+
+=== Importing the libraries
+
+[tabs]
+====
+
+// end::excludeDownstream[]
+Cordova::
++
+--
+
+. Install the link:https://www.npmjs.com/package/@aerogear/security[AeroGear Security] package from link:https://www.npmjs.com/[NPM^]
```

Comment:
This should end with a colon to be consistent with the other steps

Reply:
:+1: